examples/tokens/ERC721.vy

```diff
@@ -259,7 +259,7 @@ def safeTransferFrom(
         _from: address,
         _to: address,
         _tokenId: uint256,
-        _data: bytes[1024]=""
+        _data: bytes[1024]=b""
```

Comment:
Does this fail with the changes in this PR?

Regardless, could probably move this to a general cleanup PR